layouts/_default/baseof.html

```diff
@@ -19,6 +19,8 @@
         | resources.ToCSS (dict "outputStyle" "compressed" "enableSourceMap" (not hugo.IsProduction) "includePaths" (slice "."))
         | fingerprint -}}
     <link href="{{ $style.RelPermalink }}" rel="stylesheet">
+    {{ $script := resources.Get "js/copyable.js" | fingerprint -}}
+    <script type="module" src="{{ $script.RelPermalink }}"></script>
```

Comment:
I the `<script>` tag here so that the code can easily be used on all pages, although it is only used on the Downloads page currently.